crates/polars-lazy/src/frame/mod.rs

```diff
@@ -1047,20 +1168,38 @@ impl LazyFrame {
         feature = "csv",
         feature = "json",
     ))]
-    fn sink(mut self, payload: SinkType, msg_alternative: &str) -> Result<(), PolarsError> {
+    fn sink(
+        mut self,
+        payload: SinkType,
+        mut engine: Engine,
+        msg_alternative: &str,
+    ) -> Result<(), PolarsError> {
+        // Default to the old streaming engine.
```

Comment:
This I want to change actually this release. :) Let's already default to streaming.